client/src/components/AddBookModal/AddBookModal.tsx

```diff
@@ -39,22 +41,30 @@ const AddBookModal: React.FC<AddBookModalProps> = ({closeModalFunc, addBookFunct
   // }, [])
 
   const handleChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement>): void => {
-    if(e.target.name === 'availableToBorrow') setUserInput({...userInput, availableToBorrow: !userInput.availableToBorrow })
-    if(e.target.name === 'starRead') setUserInput({...userInput, starRead: !userInput.availableToBorrow })
-    setUserInput(prevState => {
-      return {
-        ...prevState,
-        [e.target.name]: e.target.value
-      }
-    })
+    if(e.target.name === 'availableToBorrow') {
+      setUserInput(prev => {
+            return {
+              ...prev,
+              availableToLend: !prev.availableToBorrow
+            }
+        })
+    } else if(e.target.name === 'starRead') setUserInput({...userInput, starRead: !userInput.starRead })
+    else {
+      setUserInput(prevState => {
+        return {
+          ...prevState,
+          [e.target.name]: e.target.value
+        }
+      })
+    }
   }
 
   let content; 
 
   if(modalContent === 1) {
```

Comment:
I came across this recently: https://antongunnarsson.com/react-component-code-smells/#jsx-returns . It's a useful article generally, even if I don't agree with everything, but thought it might be worth considering his point on jsx returns for this section of code.